api/app/signals/apps/api/templates/api/swagger/openapi.yaml

```diff
@@ -19,57 +19,56 @@ paths:
     get:
       description: Signals list endpoint
       parameters:
-        - name: "created_before"
+        - name: id
           in: query
           description: >-
-            Filter returns only signals created before specified datetime. Note
-            the date time string should be in ISO 8601 format and URL encoded.
+            Filter Signals on a given id
           schema:
-            type: string
-            format: date-time
+            type: integer
+            example: 123456789
           required: false
-        - name: "created_after"
+        - name: address_text
           in: query
           description: >-
-            Filter returns only signals created after specified datetime. Note
-            the date time string should be in ISO 8601 format and URL encoded.
+            Filter Signals located on a (part) of an address (case insensitive)
           schema:
             type: string
-            format: date-time
+            example: 'Amstel 1'
           required: false
-        - name: "updated_before"
+        - name: area_code
           in: query
           description: >-
-            Filter returns only signals updated before specified datetime. Note
-            the date time string should be in ISO 8601 format and URL encoded.
+            Filter signals according to the currently assigned location area_code, accepts
+            a single area_code. area_type_code is required if area_code is present.
           schema:
             type: string
-            format: date-time
+            example: 'ASC'
           required: false
-        - name: updated_after
+        - name: area_type_code
           in: query
           description: >-
-            Filter returns only signals updated after specified datetime. Note
-            the date time string should be in ISO 8601 format and URL encoded.
+            Filter signals according to the currently assigned location area_type_code, accepts
+            a single area_type_code. area_code is required if area_type_code is present.
           schema:
             type: string
-            format: date-time
+            example: 'district'
           required: false
-        - name: status
+        - name: buurt_code
```

Comment:
Candidate for removal, not sure this filter option is used anymore --- the buurtcode is not always determined. (consider marking this as deprecated)